test/vbmctl/main.go

```diff
@@ -175,20 +175,14 @@ func CreateLibvirtVM(conn *libvirt.Connect, name, networkName, macAddress string
 		return err
 	}
 
-	dom, err := conn.DomainDefineXML(vmCfg)
+	_, err = conn.DomainDefineXML(vmCfg)
 
 	if err != nil {
 		fmt.Println("Failed to define domain")
 		fmt.Printf("Error occurred: %v\n", err)
 		return err
 	}
 
-	if err := dom.Create(); err != nil {
```

Comment:
From what I understand, this would start the VM. When combined with network boot (see the template below), we could end up booting IPA in this way even if virtualmedia was specified.

Reply:
Maybe not but it doesn't hurt I would say. If we keep it, the VM would still be created and try to boot from the empty disk.